### DISCUSSION TITLE
What is the relationship between engula and sekas?

### DISCUSSION THREAD

**Discussion by xpromache** [w41ter/sekas#66] [General]
Hello,

I was looking for a KV datastore, I found engula which has no code commit in the last two years and then I found this project which seems to be a continuation? At least these are the same:
https://github.com/engula/engula/blob/main/docs/design.md
https://github.com/w41ter/sekas/blob/main/docs/design.md

Can you please clarify?
It would be best of course on the engula website to say the project is discontinued and point to this one....

**Comment by w41ter** [comment]
Englua was a project from my previous company, but development was halted due to a shift in strategic goals. Sekas is a project I continued to develop based on Englua, incorporating some of my ideas and making numerous code adjustments. Sekas aims to provide a highly scalable, strongly consistent key-value store that supports ACID interactive transactions. There are still many unfinished parts, which are currently under heavy development.

**Reply by xpromache** [reply]
Since they are not developing anymore, can't you convince them to put a link on their page redirecting to your project?
Or maybe just close down the site to avoid confusing people.

**Reply by w41ter** [reply]
Ok, I'll give it a try.
